### ISSUE TITLE
cancel

### ISSUE THREAD

**Issue by fallrain114** [xuannianz/EfficientDet#97]

**Comment by fallrain114** [comment]
i used `python3 train.py --snapshot imagenet --phi 0 --gpu 0 --random-transform --compute-val-loss --freeze-backbone --batch-size 32 --steps 1000 csv train.csv classes.csv` to train it on my own dataset
